images/contestant/devtools.nix

```diff
@@ -0,0 +1,36 @@
+{ pkgs, ... }:
+
+{
+  environment.systemPackages = with pkgs; [
+    vscode
+    neovim-gtk #replaces vim-gtk3?
+    # emacs
+    # gedit
+    # nano
+    # codeblocks
+    # geany
+    # kate
+    neovim
+    # jetbrains.pycharm-community
+    # jetbrains.idea-community
+    # jetbrains.clion
```

Comment:
Are the jetbrains things commented because development and faster installs or is it broken?

Reply:
No takes up a lot of space (which I lack)